src/js/view/controls/components/volumetooltipicon.js

```diff
@@ -0,0 +1,85 @@
+import TooltipIcon from 'view/controls/components/tooltipicon';
+import Slider from 'view/controls/components/slider';
+import UI from 'utils/ui';
+import { setAttribute, toggleClass } from 'utils/dom';
+
+class VolumeSlider extends Slider {
+    constructor(orientation, label, styleElement, callbacks) {
+        const className = orientation === 'vertical' ? 'jw-slider-volume jw-volume-tip' : 'jw-slider-volume';
+        super(className, orientation);
+        this.setup();
+
+        this.element().classList.remove('jw-background-color');
+
+        setAttribute(styleElement, 'tabindex', '0');
+        setAttribute(styleElement, 'aria-label', label);
+        setAttribute(styleElement, 'aria-orientation', orientation);
+        setAttribute(styleElement, 'aria-valuemin', 0);
+        setAttribute(styleElement, 'aria-valuemax', 100);
+        setAttribute(styleElement, 'role', 'slider');
+
+        // Apply a click interaction listener to help with focus styling
+        const { openTooltip, closeTooltip } = callbacks;
+        this.uiOver = new UI(styleElement)
+            .on('click', function() {}, callbacks)
+            .on('over', openTooltip, callbacks)
```

Comment:
seems odd to pass in tooltip functions (`openTooltip` and closeTooltip). Since slider is eventable, could we trigger events ?

Reply:
it would also decouple the volumeSlider from `VolumeTooltipIcon`